kubernetes-model/Gopkg.lock

```diff
@@ -239,12 +238,11 @@
     "k8s.io/api/authentication/v1",
     "k8s.io/api/authorization/v1",
     "k8s.io/api/autoscaling/v1",
-    "k8s.io/api/autoscaling/v2beta1",
+    "k8s.io/api/autoscaling/v2beta2",
     "k8s.io/api/batch/v1",
     "k8s.io/api/batch/v1beta1",
     "k8s.io/api/certificates/v1beta1",
     "k8s.io/api/coordination/v1",
-    "k8s.io/api/coordination/v1beta1",
```

Comment:
Why this? Was it an unused package?

Reply:
No idea (but probably), this is an autogenerated file by `dep ensure`.